scripts/mysqld_safe.sh

```diff
@@ -21,9 +21,11 @@ mysqld_ld_library_path=
 flush_caches=0
 numa_interleave=0
 wsrep_on=0
+simulate=0
 
 # Initial logging status: error log is not open, and not using syslog
 logging=init
+mysqld_safe_logging=init
```

Comment:
Do we really need this variable? Can we instead do "[ $simulate -eq 0 ] && return" from log_generic()?